binary_transparency/firmware/api/witness.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+const (
+	//WitnessGetCheckpoint is the path of the URL to get witness checkpoint.
+	WitnessGetCheckpoint = "ft/witness/v0/get-w-checkpoint"
+)
+
+// WCheckpoint commits to the state of the log.
+type WCheckpoint struct {
```

Comment:
I'm not sure this is needed, I think a `LogCheckpoint` will be sufficient. We'll want the log signature to be passed around when that is implemented.

Reply:
Agree, I will also add a comment to refer to LogCheckpoint for reference to what is returned and been symentically similar interface